src/app/home/home.component.ts

```diff
@@ -44,6 +44,9 @@ export class HomeComponent implements OnInit {
     this.currentLocationsService.getData().subscribe(data => {
```

Comment:
getData()
.filter(data =>  data &&  data._body &&  data._body.response && data._body.response.locations)
.map(data => data._body.response.locations)
.map(locations=>locations.map(location => {
 location.long_title_formatted = location.long_title.replace(",", "_");
return location;
}))
.subscribe(locations => {
 this.instructionText = "Please select a location below:";
        this.currentLocations = locations;
})